client/spago.dhall

```diff
@@ -0,0 +1,57 @@
+{-
+Welcome to a Spago project!
+You can edit this file as you like.
+-}
+{ name = "try-purescript"
+, dependencies =
+  [ "arrays"
+  , "bifunctors"
+  , "console"
+  , "const"
+  , "contravariant"
+  , "control"
+  , "distributive"
+  , "effect"
+  , "either"
+  , "enums"
+  , "exceptions"
+  , "exists"
+  , "foldable-traversable"
+  , "foreign"
+  , "foreign-generic"
+  , "foreign-object"
+  , "free"
+  , "functions"
+  , "functors"
+  , "generics-rep"
+  , "globals"
+  , "identity"
+  , "integers"
+  , "jquery"
+  , "js-timers"
+  , "lazy"
+  , "math"
+  , "maybe"
+  , "ordered-collections"
+  , "parallel"
+  , "prelude"
+  , "profunctor"
+  , "proxy"
+  , "psci-support"
+  , "quickcheck"
+  , "random"
+  , "refs"
+  , "semirings"
+  , "st"
+  , "strings"
+  , "tailrec"
+  , "transformers"
+  , "tuples"
+  , "typelevel-prelude"
+  , "unfoldable"
+  , "validation"
+  , "web-html"
+  ]
```

Comment:
Are all these dependencies actually needed for the client? Or were these just the package set dependencies for what you could compile against?

Reply:
This list of dependencies was generated by `spago init` from the existing `bower.json` file. They might not all be needed, I'm not sure.